README.md

```diff
@@ -95,3 +95,24 @@ List of every HackerOoT contributors, from most recent to oldest contribution:
 - ariahiro64
 - ghost
 - krm01
+- Sauraen
+
+This repository contains sauraen's improved animation morphing code, which can be found [here](https://github.com/sauraen/OoTAnimInterp/blob/master/animinterp.c). The following is the copyright notice pertaining to said code:
+
+```
+Copyright (C) 2021 Sauraen
+You may redistribute, create derivative works of, and otherwise copy this
+software, subject to the following conditions:
+- If the software or modified versions of it are distributed in source form,
+  it must retain this copyright notice and list of conditions
+- Sauraen must be credited (e.g. in the Special Thanks) of any project using
+  this software or modified versions of it
+
+This software does not contain any information derived from the Gigaleak.
+
+See README.md for more general information about OoT's math conventions and the
+design of this patch. Algorithms are modified from:
+https://en.wikipedia.org/wiki/Conversion_between_quaternions_and_Euler_angles
+http://www.euclideanspace.com/maths/algebra/realNormedAlgebra/quaternions/slerp/
+http://www.euclideanspace.com/maths/geometry/rotations/conversions/quaternionToEuler/Quaternions.pdf
```

Comment:
This refers to the original repo's readme, so either link to there or reword/remove this

Reply:
Fixed. Thanks